client/src/utils/vtkCustomCubeAxesActor.js

```diff
@@ -0,0 +1,83 @@
+import macro from "@kitware/vtk.js/macros";
+import vtkCubeAxesActor from "@kitware/vtk.js/Rendering/Core/CubeAxesActor";
+import * as d3 from "d3-scale";
+
+const facesToDraw = [false, false, false, false, false, true];
+const edgesToDraw = [0, 0, 0, 0, 1, 2, 2, 1, 0, 0, 0, 0];
+
+function vtkCustomCubeAxesActor(publicAPI, model) {
+  model.classHierarchy.push("vtkCustomCubeAxesActor");
+  model.cubeAxesActorHelper = vtkCubeAxesActor.newCubeAxesActorHelper();
+  global.cubeModel = model;
```

Comment:
why global?

Reply:
Started from an example, just forgot to do some final cleanup.